src/pudl/transform/eia860.py

```diff
@@ -186,6 +186,25 @@ def _core_eia860__ownership(raw_eia860__ownership: pd.DataFrame) -> pd.DataFrame
     return own_df
 
 
+def fix_boolean_columns(
```

Comment:
this change was really preemetive / i got the wind and solar mixed up. the solar is the one with basically all new boolean columns that could just use this same process